src/components/OSCALPopover.js

```diff
@@ -0,0 +1,91 @@
+import React, { useRef } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Popover from "@material-ui/core/Popover";
+import { Grid, TextField } from "@material-ui/core";
+import Typography from "@material-ui/core/Typography";
+import OSCALEditableFieldActions from "./OSCALEditableFieldActions";
+
+const useStyles = makeStyles((theme) => ({
+  ControlProseEditor: {
+    padding: theme.spacing(2),
+    width: 500,
+  },
+}));
+
+export default function OSCALPopover(props) {
+  const classes = useStyles();
+  const implementationReference = useRef("Reference to control implementation");
+  const descriptionReference = useRef("Reference to control description");
+
+  return (
+    <Popover
+      open={props.anchorEl}
+      onClose={() => {
+        props.setAnchorEl(null);
+      }}
+      anchorEl={props.anchorEl}
+      anchorOrigin={{
+        vertical: "bottom",
+        horizontal: "center",
+      }}
+    >
+      <Grid container className={classes.ControlProseEditor} xs={12}>
+        <Typography variant="h6">
+          Editing control {props.controlId}, statement {props.statementId}
```

Comment:
If we take this UX/UI approach then:

```suggestion
          Implementation of Statement '{props.statementId}' for Control '{props.controlId}'
```

However, as discussed in meetings it might make more sense to bring focus to the prose as already displayed on the page with inline input fields, in which case this might not be needed.